src/python/qeqiskit/backend/backend.py

```diff
@@ -31,14 +30,14 @@ def __init__(
         optimization_level: Optional[int] = 0,
         retry_delay_seconds: Optional[int] = 60,
         retry_timeout_seconds: Optional[int] = 86400,
+        n_samples_for_calibration: Optional[int] = None,
```

Comment:
This is nitpicky, but can we change this to `n_samples_for_readout_calibration` just so that it is more explicit?

Reply:
Done in https://github.com/zapatacomputing/qe-qiskit/pull/53/commits/48e2f0d321020ccebb6410f757a79b7469dd6ec8